src/mlpack/bindings/python/mlpack/io_util.hpp

```diff
@@ -163,6 +163,34 @@ inline void EnableTimers()
   Timer::EnableTiming();
 }
 
+/**
+ * Sanity Check.
+ */
+void SanityChecks()
+{
+  std::map<std::string, util::ParamData>::iterator itr;
+  for (itr = IO::Parameters().begin(); itr != IO::Parameters().end(); ++itr)
+  {
+    std::string paramName = itr->first;
+    std::string paramType = itr->second.cppType;
+    if (paramType == "arma::mat")
```

Comment:
Don't forget about `arma::Mat<size_t>`, `arma::Row<size_t>`, `arma::Col<size_t>`, and then the 'categorical matrix' type which is either `std::tuple<data::DatasetInfo, arma::mat>` or `std::tuple<arma::mat, data::DatasetInfo>` (I can't remember which---you can check `src/mlpack/core/util/param.hpp` to see how it is defined in `PARAM_MATRIX_WITH_INFO()`. :+1:)

Reply:
done.